autosklearn/automl.py

```diff
@@ -1612,9 +1635,7 @@ def _load_best_individual_model(self):
         even though no ensemble was found by ensemble builder.
         """
         # We also require that the model is fit and a task is defined
-        # The ensemble size must also be greater than 1, else it means
-        # that the user intentionally does not want an ensemble
-        if not self._task or self._ensemble_size < 1:
+        if not self._task or self._ensemble_class is None:
```

Comment:
Seems weird that `_load_best_individual_model` would need the `_ensemble_class` specified.

Reply:
This was the equivalent behavior to what was removed in https://github.com/automl/auto-sklearn/pull/1479. I'll remove it later here as well.

Reply:
Changed this during a rebase. The behavior is as expected now.